src/registered.js

```diff
@@ -58,26 +58,34 @@ module.exports = async function (request, h) {
     codeHosting: [{
       url: url
     }],
+    alternativeId: ipa,
   };
 
   const apiURL = config.apiURL.replace(/\/$/, '');
 
   try {
-    const res = await fetch(`${apiURL}/publishers`, {
-      method: 'POST',
-      body: JSON.stringify(apiPayload),
+    const getPublisherResp = await fetch(`${apiURL}/publishers/${ipa}`, {
+      method: 'GET',
       headers: {'Content-Type': 'application/json', 'Authorization': `Bearer ${apiPasetoKey}`},
     });
 
-    const data = await res.json();
+    const publisher = await getPublisherResp.json();
 
-    if (!res.ok) {
-      throw data.validationErrors?.map(error => `
-        Valore non valido: <strong>${error.value}</strong> per il campo: <strong>${error.field}</strong>
-        con la regola: <strong>${error.rule}</strong><br>
-      `).join('') || data.detail;
+    switch (getPublisherResp.status) {
+    case 200:
+      if (publisher.alternativeId !== ipa) {
+        throw new Error('Si è verificato un errore imprevisto durante la verifica dell\'esistenza del publisher');
+      }
+
+      await updateExistingPublisher(apiURL, apiPayload, apiPasetoKey, publisher.id);
```

Comment:
This uses the same `apiPayload` of the `POST`, but we need to *add* the new codeHosting url to the current one(s).

Also we probably don't want to update `description` because, even if it changes in IndicePA between two onboardings, it could overwrite an hypothetical manual change to the Publisher description in the API.

`alternativeId` and `pec` are keys (or part of it), so they don't change by definition. We can leave those out.

Reply:
So we should only update email and add current codeHosting that already exist, right?